deploy-board/deploy_board/settings.py

```diff
@@ -343,6 +343,8 @@
     # TODO: The model for host type should be rebuilt based on a unique abstract factor such as ec2 instance type, for now we should keep expected behavior.
     DEFAULT_CMP_HOST_TYPE = 'EbsComputeLo(Recommended)'
     DEFAULT_CMP_ARM_HOST_TYPE = 'EbsComputeXLoArm'
+    DEFAULT_HOST_TYPE_STATUS = 'ONBOARDING'
```

Comment:
can we make an enum of host type status?

Reply:
sure. Do you mean a enum in Rodimus so I can get rid of this line?

Reply:
Removed this line and created enum